doc/user/content/concepts/reaction-time.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Reaction Time, Freshness, and Query Latency
+description: "Learn about indexes in Materialize."
+menu:
+  main:
+    parent: concepts
+    weight: 1
+    identifier: 'concepts-reaction-time'
+---
+
+In operational data systems, the performance and responsiveness of queries depend not only on how fast a query runs, but also on how current the underlying data is. This page introduces three foundational concepts for evaluating and understanding system responsiveness in Materialize:
+
+* **Freshness**: how quickly new data becomes queryable,
```

Comment:
Should we use the same wording as we have in the summary below?

Reply:
Done